### PR TITLE
Release Notes for 2.6.3

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Draft of release notes for 2.6.3

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: _data/versions.yml [hunk 1]
```diff
@@ -1,4 +1,55 @@
 v2.6:
+- title: v2.6.3
+  note: |
+    28 November 2017
+    Calico v2.6.3 addresses several outstanding common vulnerabilities and exposures (CVE).
```
Comment (caseydavenport) [inline _data/versions.yml]: Looks like there are bullet points under this title, which isn't quite right - the things under this aren't CVEs

File: _data/versions.yml [hunk 2]
```diff
@@ -1,4 +1,55 @@
...
+    - A new node controller for Kubernetes deployments clears data associated with deleted nodes from the Calico datastore, preventing conflicts that can lead to crash loops.
```
Comment (caseydavenport) [inline _data/versions.yml]: Is this really all that's changed? I bet there is more if we run the release-note script (though maybe not!)

Comment (bcreane) [reply _data/versions.yml]: Yeah, I ran the release-note script and this is basically what showed up.

File: _data/versions.yml (hunk 1 above)
Comment (emanic) [reply _data/versions.yml]: I would say I agree... @bcreane this sentence is pretty broad, any further details on these? Either way, we could make it it's own bullet

File: _data/versions.yml (hunk 2 above)
Comment (emanic) [reply _data/versions.yml]: Oh, I also heard that it might have adjusted the confd log level?

Comment (bcreane) [reply _data/versions.yml]: guess we skipped setting the milestone for some of those last minute fixes, which leaves `generate-release-notes.py` empty handed.

Comment (bcreane) [reply _data/versions.yml]: The confd debug->info went in as part of a much larger commit, unfortunately: https://github.com/projectcalico/calico/pull/1395. @fasaxc and I discussed whether to break that out into a separate PR, but skipped past it. So we'll have to generate a note for this manually.

**Pull request merged**